### DISCUSSION TITLE
Blender Launcher blank

### DISCUSSION THREAD

**Discussion by narbartor** [Victor-IX/Blender-Launcher-V2#170] [Q&A]
I recently updated both my computer and my graphics driver, and afterwards I see that my Blender Launcher is now just Blank whenever I open it.

At first, blender launcher opens on start up and for some reason asks me where to store Blender versions, as if it were my very first time opening. But once I try restarting the Launcher, I just get a blank window

![image](https://github.com/user-attachments/assets/947da4f6-3760-4424-a506-1d2cb4a42712)

**Comment by Victor-IX** [comment]
If you are on Windows can you go in `%localappdata%/Blender Launcher`, you will have a `Blender Launcher.ini` and a `Blender Launcher.log` files, can you link them both here?

**Reply by narbartor** [reply]
[Blender Launcher.log](https://github.com/user-attachments/files/17806971/Blender.Launcher.log)
[stable_builds.json](https://github.com/user-attachments/files/17806972/stable_builds.json)

Here it is

**Reply by Victor-IX** [reply]
I need the `Blender Launcher.ini` file if possible.
What you can try is deleting this file with the Launcher completely close and reopening Blender Launcher. You will be prompted to set your Blender Launcher Library folder. Set it back to the same folder as previously and check if it fixes your issue.

**Reply by narbartor** [reply]
My apologies, I thought I uploaded it, but it seems like it doesnt show up, in the file selector, so I'll just paste the text into here

``[General]
library_folder=.
launch_minimized_to_tray=true
buildcheck_on_startup=false

[Internal]
DownloadsDailyListWidget_sorting_type=1
DownloadsExperimentalListWidget_sorting_type=1
DownloadsStableListWidget_sorting_type=1
LibraryDailyListWidget_sorting_type=1
LibraryExperimentalListWidget_sorting_type=1
LibraryStableListWidget_sorting_type=1
UserCustomListWidget_sorting_type=1
UserFavoritesListWidget_sorting_type=1
last_time_checked_utc=2024-11-12T06:25:52+08:00
tray_icon_notified=true
favorite_path=D:/Extras/Blender versions/custom/blender-4.2.0-windows-x64
``

**Reply by Victor-IX** [reply]
Are your library folders located in the same place as the Blender Launcher.exe? 
Also, did you try that?
> What you can try is deleting this file with the Launcher completely close and reopening Blender Launcher. You will be prompted to set your Blender Launcher Library folder. Set it back to the same folder as previously and check if it fixes your issue.

**Reply by zeptofine** [reply]
I remember this being an issue with trying to launch it while it's already open, but I'm not sure why the resulting launcher does not have any widgets drawn. it was very inconsistent for me

**Reply by narbartor** [reply]
> Are your library folders located in the same place as the Blender Launcher.exe? 

This is what my blender launcher folder looks like. I put my blender versions in the "custom" folder
![image](https://github.com/user-attachments/assets/9d55414e-ec1f-44c9-973e-4d6d2cb8722c)


> What you can try is deleting this file with the Launcher completely close and reopening Blender Launcher. You will be prompted to set > your Blender Launcher Library folder. Set it back to the same folder as previously and check if it fixes your issue.


I tried it. I deleted my Ini file and opened blender launcher and I dont even get the option to set my library folder. Its just straight to blank window

**Reply by Victor-IX** [reply]
Can you try reinstalling the latest version of the Blender Launcher manually? Also, delete the Blender Launcher folder in `%localappdata%` to start from a clean state.
